src/context/meta-tags-context.php

```diff
@@ -36,7 +36,6 @@
  * @property int         $site_user_id
  * @property string      $site_represents
  * @property array|false $site_represents_reference
- * @property bool        $breadcrumbs_enabled
```

Comment:
Should $breadcrumbs_enabled also be removed from the value object in src/surfaces/values/meta.php?

Reply:
If so, it should also be removed from https://github.com/Yoast/developer-docs/blob/master/docs/customization/apis/surfaces-api.md (cc @jcomack)

Reply:
Done in 25a9aba07e05acbf4c3ed7db1f0f6646d8140a95
@jcomack can you update the docs?